src/features/MonthView/styles.css

```diff
@@ -0,0 +1,28 @@
+.month-cell-wrapper {
+  display: flex;
+  flex-wrap: wrap;
+  align-items: stretch;
+  height: 84vh;
+}
+
+.month-cell {
+  display: flex;
+  flex: 1 0 calc(14.28% - 10px);
```

Comment:
14.28% is a scary number and I didn't get it right away what it means. If that if 100 / 7 then state it like that or even better move it to the constants with a descriptive name and then use it here